lib/lens/trace.rb

```diff
@@ -21,45 +21,42 @@ def add(event)
 
     def complete(event)
       formatted_data = Lens::EventFormatter.new(event, @data, @gc_statistics.total_time).json_formatted
-      log(formatted_data)
       send(formatted_data)
       Thread.current[:__lens_trace] = nil
     end
 
-  private
+    private
 
     def send(data)
-      log(data)
       Worker.instance.push(data)
     end
-
-    def log(data)
-      Rails.logger.info "all [LENS] >>> #{data}" if verbose?
-    end
-
-    def verbose?
-      false
-    end
   end
 
   class << Trace
     def process(*args)
-      name, _started, _finished, id, _data = [*args]
+      name, _started, _finished, id, _data = args
```

Comment:
Useless assignment to variable - `name`. Use `_` or `_name` as a variable name to indicate that it won't be used.